### DISCUSSION TITLE
Local aplication call not working

### DISCUSSION THREAD

**Discussion by Omaeda** [hoothin/SearchJumper#149] [Q&A] [answered]
![imagen](https://github.com/user-attachments/assets/62efec1a-85ae-4013-8380-1cf84f512fa5)

**Comment by hoothin** [comment, marked as answer]
Windows only.

**Comment by hoothin** [comment]
You need to add URI scheme manually for calling local app on linux.
https://unix.stackexchange.com/questions/497146/create-a-custom-url-protocol-handler
